### PR TITLE
Add filter by a category, filter by confidence, display findings from a specific time

### PR DESCRIPTION
**Pull request by adamkvitek**
# Thanks for improving Semgrep Docs 😀

### Please ensure

- [x] A subject matter expert (SME) reviews the content
- [x] A technical writer reviews the content or PR
- [x] This change has no security implications or else you have pinged the security team
- [ ] Redirects are added if the PR changes page URLs

### CODE REVIEW

File: docs/semgrep-app/findings.md [hunk 1]
```diff
@@ -95,42 +95,72 @@ The Findings page consists of:
...
-| **Rules**        | Filter by rules or rulesets that are included in your Rule Board. More than one rule can be selected for filtering. |
-| **Rulesets**     | Filter by name of the ruleset where rules that matched the code belong. |
+| **Status**       | Filter for different findings triage states. Refer to the [following table](#triaging-findings) to understand triage states. |
+| **Severities**    | Filter by the severity of a finding. Possible values: <ul><li>Low</li><li>Medium</li><li>High</li></ul> |
 | **Actions**     | Filter by monitoring, commenting, or blocking rules in your Rule Board. |
-| **Severity**    | Filter by the severity of a finding. Possible values: <ul><li>Low</li><li>Medium</li><li>High</li></ul> |
+| **Confidences** | Filter by indication of the rule to detect true positives. The higher the confidence the more true positives the rule may detect.  |
+| **Categories**  |  Filter by various rule categories, such as security, or best practice rules. More than one category can be selected for filtering. |
+| **Branches**     | Filter by findings in different Git branches. More than one branch can be selected for filtering. |
```
Comment (annikalee) [inline docs/semgrep-app/findings.md:110]: We don't allow more than one yet. It's a bug we have filed

File: docs/semgrep-app/findings.md [hunk 2]
```diff
@@ -95,42 +95,72 @@ The Findings page consists of:
...
+| **Rules**        | Filter by rules or rulesets that are included in your Rule Board. More than one rule can be selected for filtering. More than one rule can be selected for filtering. |
```
Comment (annikalee) [inline docs/semgrep-app/findings.md:111]: I think we repeated "More than one rule can be selected for filtering" twice here

File: docs/semgrep-app/findings.md [hunk 3]
```diff
@@ -95,42 +95,72 @@ The Findings page consists of:
...
+| **Rulesets**     | Filter by name of the ruleset where rules that matched the code belong. More than one rule or ruleset can be selected for filtering. |
+
+You can also filter for findings reported from specific time periods or group findings by rule. For more information see
```
Comment (annikalee) [inline docs/semgrep-app/findings.md:114]: see?

File: docs/semgrep-app/findings.md [hunk 4]
```diff
@@ -95,42 +95,72 @@ The Findings page consists of:
...
+1. In Semgrep App, go to **[Findings](https://semgrep.dev/orgs/-/findings)** page.
+1. Click the <i class="fa-solid fa-calendar-days"></i> **All time** button.
+1. Select preferred time period for which to display findings. The following options are available:
+    - **All time** - This is the default option.
+    - Last 1 year
+    - Last 1 month
+    - Last 7 days
+    - Last 1 day
+
+### Grouping by rule
```
Comment (annikalee) [inline docs/semgrep-app/findings.md:135]: This is still in beta. (Needs a feature flag) Not sure if we should include it yet?

File: docs/semgrep-app/findings.md (hunk 2 above)
Comment (adamkvitek) [reply docs/semgrep-app/findings.md:111]:
```suggestion
| **Rules**        | Filter by rules or rulesets that are included in your Rule Board. More than one rule can be selected for filtering. |
```

File: docs/semgrep-app/findings.md (hunk 3 above)
Comment (adamkvitek) [reply docs/semgrep-app/findings.md:114]:
```suggestion
You can also filter for findings reported from specific time periods or group findings by rule. See [Displaying findings reported in a specific time](/semgrep-app/findings/#displaying-findings-reported-in-a-specific-time) and [Group by rule](/semgrep-app/findings/#grouping-by-rule) sections.
```

File: docs/semgrep-app/findings.md (hunk 1 above)
Comment (adamkvitek) [reply docs/semgrep-app/findings.md:110]:
```suggestion
| **Branches**     | Filter by findings in different Git branches. |
```

File: docs/semgrep-app/findings.md (hunk 4 above)
Comment (adamkvitek) [reply docs/semgrep-app/findings.md:135]: @annikalee ok...

Comment (adamkvitek) [reply docs/semgrep-app/findings.md:135]: Solutions:
1. Remove this part of the documentation and keep it in another PR.
2. Mark this feature as a beta for specific customers who may request it.

Comment (pabloest) [reply docs/semgrep-app/findings.md:135]: @adamkvitek Channeling my inner Raghav here, I’d go for #2 but you may want to confirm with Chinmay.

**Review by pabloest — APPROVED** [review]
I replied in one of the discussions. I haven’t used this feature directly myself so keep that in mind.

File: docs/semgrep-app/findings.md (hunk 4 above)
Comment (adamkvitek) [reply docs/semgrep-app/findings.md:135]: @neo413001 we were thinking with @s-santillan that solution 1. (see the conversation above in this thread) might be better for now. Should we mark this section about grouping by rule as beta or move it to another PR and wait until the feature is generally out for everyone? I think that if we mark it as beta we need to say that users need to contact us to get this beta feature.

File: docs/semgrep-app/findings.md [hunk 5]
```diff
@@ -95,36 +95,52 @@ The Findings page consists of:
...
 
-1. Click the filter criteria drop-down box.
-2. Enable displayed checkbox or write in a field value by which the findings are filtered.
-3. The page then refreshes to reflect the additional criteria.
-4. Additional values may be selected to further refine your filter.
+To filter through findings:
+
+1. Click the filter criteria.
+1. The page then refreshes to reflect the additional criteria.
+1. Optional: To futher refine your filter, select additional filters.
```
Comment (adamkvitek) [inline docs/semgrep-app/findings.md:120]:
```suggestion
1. Click the filter criteria. The page then refreshes to reflect the additional criteria.
1. Optional: To further refine your filter, select additional filters.
```

File: docs/semgrep-app/findings.md [hunk 6]
```diff
@@ -95,36 +95,51 @@ The Findings page consists of:
...
 | **Actions**     | Filter by monitoring, commenting, or blocking rules in your Rule Board. |
-| **Severity**    | Filter by the severity of a finding. Possible values: <ul><li>Low</li><li>Medium</li><li>High</li></ul> |
+| **Confidences** | Filter by indication of the rule to detect true positives. The higher the confidence the more true positives the rule may detect.  |
+| **Categories**  |  Filter by various rule categories, such as security, or best practice rules. More than one category can be selected for filtering. |
+| **Branches**     | Filter by findings in different Git branches. |
+| **Rules**        | Filter by rules or rulesets that are included in your Rule Board. More than one rule can be selected for filtering. |
+| **Rulesets**     | Filter by name of the ruleset where rules that matched the code belong. More than one rule or ruleset can be selected for filtering. |
 
-To filter through all findings:
+You can also filter for findings reported from specific time periods or group findings by rule. See [Displaying findings reported in a specific time](/semgrep-app/findings/#displaying-findings-reported-in-a-specific-time).
```
Comment (adamkvitek) [inline docs/semgrep-app/findings.md:114]:
```suggestion
You can also filter for findings reported from specific time periods. See [Displaying findings reported in a specific time](/semgrep-app/findings/#displaying-findings-reported-in-a-specific-time).
```

**Pull request merged**